### DISCUSSION TITLE
Fajer TV 1

### DISCUSSION THREAD

**Discussion by l-ben** [iptv-org/community#711] [Q&A]

**Comment by BellezaEmporium** [comment]
http://tv.alfajertv.com/MY_VIDEO.webm

**Comment by Alstruit** [comment]
HD 1 [Geo-blocked]
https://vstream2.hadara.ps:8443/AlfajertvHDFull_web/web/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9MTAvMS8yMDIyIDQ6MTA6MTMgQU0maGFzaF92YWx1ZT1oWGxuWmFUdFVFQjZQZjFYd1hxaGtRPT0mdmFsaWRtaW51dGVzPTE4MCZpZD0zZDAxZjQ1MGU5MDQ5NjRiYTczZjJhMDgwMGQxMTAxNA==

1 [Geo-blocked]
https://vstream4.hadara.ps:8443/AlfajertvHD1_web/web/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9MTAvMS8yMDIyIDQ6MTA6NDEgQU0maGFzaF92YWx1ZT0rZldKWjA3cklWMjlRaTZ3d215OHlRPT0mdmFsaWRtaW51dGVzPTE4MCZpZD02MDNmOWQxNmExOWYzNTE4NWI3YWY5Zjc3MjNiNGU2Mg==

International
https://vstream3.hadara.ps:8443/AlfajertvUN_web/web/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9MTAvMS8yMDIyIDQ6MDk6MjEgQU0maGFzaF92YWx1ZT1YbU5RMGRQYVRhaFNmYTVtQVFPd3lRPT0mdmFsaWRtaW51dGVzPTE4MCZpZD1hNjRmN2EyMDRlNDc0MTRjNDU3MGVjNGU5YTI0NjMyYg==

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ps.m3u#L12-L13
